auth/store.go

```diff
@@ -1159,11 +1159,12 @@ func (as *authStore) AuthInfoFromTLS(ctx context.Context) (ai *AuthInfo) {
 
 	tlsInfo := peer.AuthInfo.(credentials.TLSInfo)
 	for _, chains := range tlsInfo.State.VerifiedChains {
-		if len(chains) < 1 {
+		cn := chains[0].Subject.CommonName
```

Comment:
This would panic when `len(chains) < 1`?

Reply:
nice catch, thanks